gordo_components/model/anomaly/diff.py

```diff
@@ -297,6 +297,7 @@ def anomaly(
             # Dataframe of % abs_diff is of the thresholds
             anomaly_confidence_scores = pd.DataFrame(
                 confidence,
+                index=tag_anomaly_scaled.index,
```

Comment:
this came a bit out of the blue, is it related to the other stuff or just another bug you squashed while walking around?

Reply:
Yessir, there be monsters lurking in this detector; what with all the strange calculations. :sweat_smile: